components/OnlineBattleCard/index.tsx

```diff
@@ -25,6 +25,10 @@ import SilverIcon from "~/assets/img/Silver";
 import FadeIn from "~/components/Common/FadeIn";
 
 const OnlineBattleCard = ({ data }: Props) => {
+  const battleIsOver =
+    data.eventEndDate === ""
+      ? false
+      : dayjs().valueOf() > dayjs(data.eventEndDate).valueOf();
```

Comment:
what about dates that are same date with endDate? shouldnt the condition be >= ?

Reply:
> what about dates that are same date with endDate? shouldnt the condition be >= ?

Makes sense, I'll change it to >=